app/both/collections/residencies.js

```diff
@@ -137,3 +145,10 @@ Residencies.after.remove(function (userId, residency) {
     entityId: residency._id,
   })
 });
+
+function validateDate(moveIn, moveOut){
+  const moveOutDate = new Date(moveOut).getTime();
+  const moveInDate = new Date(moveIn).getTime();
+  if(!isNaN(moveOutDate) && moveOutDate>0 && moveInDate > moveOutDate) return false
```

Comment:
Does this allow residencies to *not* have departure date?

Reply:
yes